lens/phrase.mli

```diff
@@ -90,6 +90,12 @@ module Option : sig
 
   (** Construct an in expression phrase option  *)
   val in_expr : string list -> Value.t list list -> t
+
+(** Calculate the vale of an expression given a lookup function for variables. Defaults to true if the phrase is [None]. *)
```

Comment:
vale -> value.  Also, maybe split comment into two lines to stay within 70-80 columns.

Reply:
Changed